benches/benches/set/mem.rs

```diff
@@ -114,16 +106,28 @@ pub fn run(c: &mut Criterion) {
     }
     mem_mcp.finish();
 
-    run_group_ref(
-        &mut c.benchmark_group("mcpi"),
-        "mcpi",
-        VmBench::new(op::mcpi(0x10, RegId::ZERO, Imm12::MAX.to_u16()))
-            .with_prepare_script(vec![
-                op::movi(0x11, Imm12::MAX.to_u16() as u32),
-                op::aloc(0x11),
-                op::move_(0x10, RegId::HP),
-            ]),
-    );
+    let mut mem_mcpi = c.benchmark_group("mcp");
+    for i in &linear {
+        if *i >= (1 << 12) {
+            // Too large for immedidate value
+            continue
+        }
```

Comment:
Hmm, maybe it is worth simply charging for the worst case with `1 << 12`=)

Reply:
Maybe. I'll still have this PR updating the benches to `DependentCost`, and change it to use a constant cost later when we have done actual benchmarking.